common/runmodel.pas

```diff
@@ -135,6 +136,7 @@ implementation
       Begin
         TextColor(red);
         writeln(E.Message);
+        writeln('Exception Address: ', BackTraceStrFunc(ExceptAddr));
```

Comment:
Can you add some documentation to https://github.com/watem-sedem/watem-sedem/blob/master/CONTRIBUTING.rst?